### DISCUSSION TITLE
Edit column width and row height (manually and autoadjust)

### DISCUSSION THREAD

**Discussion by JCaldas-dev** [Akascape/CTkTable#28] [Ideas]
This would probably be more useful for columns. Some columns just don't need to be the same size as other such as a column with dates would be pretty consistent with the amount of pixels needed, unlike a column with the full name of a person that varies in size.

Ideally this feature would either let us set a fixed size and/or let the column adjust by itself with some padding depending on the values found along the column.

**Comment by Akascape** [comment]
@JCaldas-dev 
You can try this method: `table.edit_column(column_num, width=30)` to set a manual width and similarly edit_row method to set manual height.

**Reply by JCaldas-dev** [reply]
perfect! but any way to auto adjust maybe?

**Reply by Akascape** [reply]
@JCaldas-dev While defining the table, pass `width=1` and It will auto adjust. Make sure the cells are not blank.

**Reply by JCaldas-dev** [reply]
Thanks a lot for the help. This is a great extension, thanks for your work
